src/js/utils/BetaOptInUtil.js

```diff
@@ -0,0 +1,85 @@
+import Util from "./Util";
+
+// SDK sets/reads this property within package config.
+const BETA_OPTIN_PROPERTY_NAME = "beta-optin";
+const DOT_NOTATION_SCHEMA_PATH = "properties.service.properties.beta-optin";
+
+const BetaOptInUtil = {
+  /**
+   * Finds beta-optin property within cosmos package config schema.
+   *
+   * @param  {Object} configSchema - cosmos package config schema
+   * @return {Boolean} Returns true if configSchema contains beta-optin property
+   */
+  isBeta(configSchema) {
+    return BetaOptInUtil.getProperty(configSchema) != null;
+  },
+
+  /**
+   * Get beta-optin property within config Object
+   *
+   * @param  {Object} configSchema - cosmos package config schema
+   * @return {Object} Returns property if found or null.
+   */
+  getProperty(configSchema) {
+    if (!Util.isObject(configSchema)) {
+      return null;
+    }
+
+    return Util.findNestedPropertyInObject(
+      configSchema,
+      DOT_NOTATION_SCHEMA_PATH
+    );
+  },
+
+  /**
+   * Removes beta-optin property within config schema
+   *
+   * @param  {Object} configSchema - cosmos package config schema
+   * @return {Object} Returns new config Object without opt-in property
+   */
+  filterProperty(configSchema) {
```

Comment:
Please rename this util to clearly state that it will "remove" (create a copy omitting the beta property) the beta property from the passed schema. #style #naming